### PR TITLE
feature/pdpdevtool-4189: vscode 'List Files' folder list cutoff

### PR DESCRIPTION
**Pull request by davidecorreu**
Improve visualization of the available folders list of the FileCabinet in vscode 'List Files' command.
Refactor of commandsMap
Fix minor output references to command names ListFilesService
Add "Setup Account" link when no defaultAuthId is available in the project when triggering 'List Files'.

Closes PDPDEVTOOL-4189

### CODE REVIEW

File: packages/vscode-extension/src/commands/ManageAccounts.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { MANAGE_ACCOUNTS, DISMISS } from '../service/TranslationKeys';
 import * as fs from 'fs';
 import * as path from 'path';
 
-const COMMAND_NAME = 'manageaccounts';
+const COMMAND_NAME = 'setupaccount';
```
Comment (antoniofruci) [inline packages/vscode-extension/src/commands/ManageAccounts.ts:14]: I think we should rename the file accordingly.

File: packages/vscode-extension/src/commands/ListFiles.ts [hunk 2]
```diff
@@ -25,7 +25,10 @@ export default class ListFiles extends BaseAction {
 	protected async execute(): Promise<void> {
 		const listFilesService = new ListFilesService(this.messageService, this.translationService, this.getRootProjectFolder());
 		try {
-			let fileCabinetFolders: FolderItem[] = await listFilesService.getListFolders();
+			let fileCabinetFolders: FolderItem[] | undefined= await listFilesService.getListFolders();
```
Comment (lfpvillegas) [inline packages/vscode-extension/src/commands/ListFiles.ts:28]: Format space

File: packages/vscode-extension/src/commandsMap.ts [hunk 3]
```diff
@@ -3,64 +3,92 @@
...
-export const commandsInfoMap: {
-	[commandName: string]: {
-		cliCommandName: string;
-		vscodeCommandName: string;
-		// not really needed, just left here as an idea
-		vscodeShortName?: string;
-	};
-} = {
+export type CommandInfo = {
+	vscodeContributedCommand: string;
```
Comment (lfpvillegas) [inline packages/vscode-extension/src/commandsMap.ts:7]: Could be just rename it to something like 
vscodeCommandId?

Or we could call this vscodeCommandName and the other one call it vscodeCommandLabel

**Review by lfpvillegas — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/src/commands/ListFiles.ts (hunk 2 above)
Comment (davidecorreu) [reply packages/vscode-extension/src/commands/ListFiles.ts]: Added.

File: packages/vscode-extension/src/commands/ManageAccounts.ts (hunk 1 above)
Comment (davidecorreu) [reply packages/vscode-extension/src/commands/ManageAccounts.ts:14]: Renamed.

File: packages/vscode-extension/src/commandsMap.ts (hunk 3 above)
Comment (davidecorreu) [reply packages/vscode-extension/src/commandsMap.ts]: Renamed to vscodeCommandId.

**Review by lfpvillegas — APPROVED** [review]

**Review by antoniofruci — APPROVED** [review]

**Pull request merged**